src/frontend/apps/desk/src/pages/mail-domains/index.tsx

```diff
@@ -5,7 +5,7 @@ import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 
 import { Box } from '@/components';
-import { MailDomainsLayout } from '@/features/mail-domains';
+import { MailDomainsLayout } from '@/features/mail-domains/index';
```

Comment:
I don't think you should have a folder `index`.

![image](https://github.com/user-attachments/assets/5b26667f-f3e6-4e32-a870-c555965d1adf)

In `index` you have `panel`, panel is used to list the `domains` so why not add panel to the folder domains?
`MailDomainsLayout` could go in domains as well IMO.
`MailDomainsContent` seems to list the mailBoxes, so let's put it in mailboxes folder.

Wdyt ?